style.css

```diff
@@ -157,3 +157,39 @@ body {
 	padding: 0 24px 0;
 	line-height: 1.5;
 }
+input[type=checkbox] {
+	border: 1px solid #b4b9be;
+	background: #fff;
+	color: #555;
+	clear: none;
+	cursor: pointer;
+	display: inline-block;
+	line-height: 0;
+	height: 16px;
+	margin: -4px 4px 0 0;
+	outline: 0;
+	padding: 0!important;
+	text-align: center;
+	vertical-align: middle;
+	width: 16px;
+	min-width: 16px;
+	-webkit-appearance: none;
+	box-shadow: inset 0 1px 2px rgba(0,0,0,.1);
+	transition: .05s border-color ease-in-out;
+}
+input[type=checkbox]:checked:before {
+    font: normal 21px/1 dashicons;
+    content: "\f147";
+    margin: -3px 0 0 -4px;
+    color: #1e8cbe;
+    float: left;
+    display: inline-block;
+    vertical-align: middle;
+    width: 16px;
+    speak: none;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+@font-face {
+    font-family:"dashicons";
+    src: url(/fonts/dashicons.ttf)
```

Comment:
Should there be a closing `}` tag here?

Reply:
Display error and minor change to code has been fixed in c54586e